### PR TITLE
#19 change application colors

### PR DESCRIPTION
**Pull request by daros10**
PR related to change app style.

App Style

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "semi": true,
+  "bracketSpacing": true,
+  "endOfLine": "lf"
+}
```
Comment (macrisguncay) [inline .prettierrc:9]: I think that you are having problems getting master changes.. why do you have these changes.?

File: README.md [hunk 2]
```diff
@@ -22,6 +23,15 @@ Run `npm install` to install the required node_modules for this project.
...
 
+## Prepare your environment for vscode
+
+Install the following extensions:
+
+- `EditorConfig for Visual Studio Code`.
+- `TSLint`
+- `Prettier - Code formatter`.
+- Go to user settings (`settings.json`) and enable formatting on save: `"editor.formatOnSave": true`.
+
```
Comment (macrisguncay) [inline README.md:34]: this changes already are in master branch

File: package.json [hunk 3]
```diff
@@ -28,8 +28,8 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.900.5",
-    "@angular/cli": "~9.0.4",
+    "@angular-devkit/build-angular": "^0.901.0",
+    "@angular/cli": "^9.1.0",
```
Comment (macrisguncay) [inline package.json:32]: is this your local update.?

File: src/app/modules/time-entries/pages/time-entries.component.scss [hunk 4]
```diff
@@ -44,3 +44,8 @@
     cursor: pointer;
   }
 }
+
+.title-card {
+  background-image: linear-gradient(to left top, #fc5630, #fe503a, #ff4a44, #ff444d, #ff3f56);
```
Comment (macrisguncay) [inline src/app/modules/time-entries/pages/time-entries.component.scss:49]: I think that it is better if we can define palette colors.

**Comment by daros10** [comment]
> Please, check your branch because you have some changes that already are merged in the master branch

The problem is that it works with the last changes that were made in the branch that is not merged to be able to apply the changes of the styles to the functionalities that are not in master.

**Comment by daros10** [comment]
> 1. You need to rebase this branch against master. Ask @macrisguncay for help, she can lend you a hand with this.
> 2. Remove the search project implementation since this is part of another ticket and we don't want to include more than one ticket per PR
> 3. There are some issues with the look and feel.
>    We are not using rounded corners for the components. Check the accepted design:
> 
> The username in the top right corner (no borders needed):
> 
> You implementation:
> Screen Shot 2020-03-31 at 23 02 36
> 
> Original design:
> Screen Shot 2020-03-31 at 23 03 33
> 
> Menu items are rectangles with no rounded corners.
> Your implementation looks like:
> Screen Shot 2020-03-31 at 23 04 34
> 
> Design looks like:
> Screen Shot 2020-03-31 at 23 05 26
> 
> ioet colors don't have any green so button colors should not be green.
> This is how clock-in button looks like:
> Screen Shot 2020-03-31 at 23 06 59
> 
> Use this set of colors as reference:
> Screen Shot 2020-03-31 at 23 07 36
> 
> You need to add some consistent colors for the texts here. The background-color should not be gray for sure.
> Screen Shot 2020-03-31 at 23 08 25
> 
> Remove rounded corners here:
> Screen Shot 2020-03-31 at 23 09 29
> 
> These buttons can not be blue, FInd a consistent color for them:
> Screen Shot 2020-03-31 at 23 10 27
> Screen Shot 2020-03-31 at 23 10 32
> 
> This looks really weird, please fix the styles:
> Screen Shot 2020-03-31 at 23 11 15
> 
> Feel free to ask @jorgecod for help. He knows a lot about styling.



> 1. You need to rebase this branch against master. Ask @macrisguncay for help, she can lend you a hand with this.
> 2. Remove the search project implementation since this is part of another ticket and we don't want to include more than one ticket per PR
> 3. There are some issues with the look and feel.
>    We are not using rounded corners for the components. Check the accepted design:
> 
> The username in the top right corner (no borders needed):
> 
> You implementation:
> Screen Shot 2020-03-31 at 23 02 36
> 
> Original design:
> Screen Shot 2020-03-31 at 23 03 33
> 
> Menu items are rectangles with no rounded corners.
> Your implementation looks like:
> Screen Shot 2020-03-31 at 23 04 34
> 
> Design looks like:
> Screen Shot 2020-03-31 at 23 05 26
> 
> ioet colors don't have any green so button colors should not be green.
> This is how clock-in button looks like:
> Screen Shot 2020-03-31 at 23 06 59
> 
> Use this set of colors as reference:
> Screen Shot 2020-03-31 at 23 07 36
> 
> You need to add some consistent colors for the texts here. The background-color should not be gray for sure.
> Screen Shot 2020-03-31 at 23 08 25
> 
> Remove rounded corners here:
> Screen Shot 2020-03-31 at 23 09 29
> 
> These buttons can not be blue, FInd a consistent color for them:
> Screen Shot 2020-03-31 at 23 10 27
> Screen Shot 2020-03-31 at 23 10 32
> 
> This looks really weird, please fix the styles:
> Screen Shot 2020-03-31 at 23 11 15
> 
> Feel free to ask @jorgecod for help. He knows a lot about styling.

I have removed all the non-existent functionality in master and have corrected the observations.

File: src/app/modules/project-management/pages/project-management.component.spec.ts [hunk 5]
```diff
@@ -44,14 +45,14 @@ describe('ProjectManagementComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ProjectManagementComponent, CreateProjectComponent, ProjectListComponent ],
-      providers: [ { provide: ProjectService, useValue: projectServiceStub }],
+      declarations: [ProjectManagementComponent, CreateProjectComponent, ProjectListComponent, FilterProjectPipe],
```
[the remark this answers is not in the document]
Comment (daros10) [reply src/app/modules/project-management/pages/project-management.component.spec.ts]: removed.

File: package.json (hunk 3 above)
Comment (daros10) [reply package.json]: removed.

File: README.md (hunk 2 above)
Comment (daros10) [reply README.md]: removed.

File: .prettierrc (hunk 1 above)
Comment (daros10) [reply .prettierrc]: removed.

File: src/app/modules/shared/components/search-project/search-project.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SearchProjectComponent } from './search-project.component';
+
+describe('SearchProjectComponent', () => {
```
[the remark this answers is not in the document]
Comment (daros10) [reply src/app/modules/shared/components/search-project/search-project.component.spec.ts]: removed.

File: src/assets/project.json [hunk 7]
```diff
@@ -12,33 +12,5 @@
     "details": "It's a good app to play Lottery",
     "status": "Inactive",
     "completed": true
-  },
-  {
-    "id": 3,
-    "name": "Facebook",
```
Comment (macrisguncay) [inline src/assets/project.json]: ?????

**Pull request closed without merging**